src/main/java/org/patternfly/component/panel/Panel.java

```
@@ -0,0 +1,122 @@
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package org.patternfly.component.panel;
+
+import org.jboss.elemento.Attachable;
+import org.patternfly.component.BaseComponentFlat;
+import org.patternfly.component.ComponentType;
+
+import elemental2.dom.HTMLDivElement;
+import elemental2.dom.MutationRecord;
+
+import static org.jboss.elemento.Elements.div;
+import static org.jboss.elemento.Elements.hr;
+import static org.patternfly.component.panel.PanelBody.panelBody;
+import static org.patternfly.component.panel.PanelFooter.panelFooter;
+import static org.patternfly.component.panel.PanelHeader.panelHeader;
+import static org.patternfly.core.Aria.labelledBy;
+import static org.patternfly.style.Classes.*;
+
+public class Panel extends BaseComponentFlat<HTMLDivElement, Panel> implements Attachable {
+    private PanelHeader header;
+    private PanelBody body;
+    private PanelFooter footer;
+
+    protected Panel() {
+        super(ComponentType.Panel, div().css(component(panel)).element());
+    }
+
+    public static Panel panel() {
+        return new Panel();
+    }
+
+    public Panel raised() {
+        return css(modifier(raised));
+    }
+
+    public Panel bordered() {
+        return css(modifier(bordered));
+    }
+
+    public Panel scrollable() {
+        return css(modifier(scrollable));
+    }
+
+    // ------------------------------------------------------ add
+
+    public Panel addHeader(String header) {
+        return add(panelHeader().textContent(header));
+    }
+
+    public Panel addHeader(PanelHeader header) {
+        return add(header);
+    }
+
+    public Panel add(PanelHeader header) {
+        if (this.header != null) {
```

I'm not sure if we should throw an exception here. I prefer not adding a second header and log an error/warning. Something like `Logger.unsupported(ComponentType, Element, String)`. Same for body and footer.